config/locales/en/placements/support/schools/users.yml

```
@@ -18,4 +18,14 @@ en:
           check:
             page_title: "Check your answers - Add user - %{school_name}"
             caption: Add user - %{organisation_name}
-            check_your_answers: Check your answers
\ No newline at end of file
+            check_your_answers: Check your answers
+          destroy:
+            user_removed: User removed
+          show:
+            remove_user: Remove user
+          remove:
+            page_title: Are you sure you want to remove this user? - %{user_name} - %{organisation_name}
+            are_you_sure: Are you sure you want to remove this user?
+            remove_user: Remove user
+            cancel: Cancel
+            warning: The user will be sent an email to tell them you removed them from %{organisation_name}
```

Not a strong opinion, but it could make the view more readable:

```suggestion
            user_will_be_sent_an_email: The user will be sent an email to tell them you removed them from %{organisation_name}
```